docs/guide/slots.md

```diff
@@ -44,7 +44,7 @@ To render a `renders_many` slot, iterate over the name of the slot:
   <% end %>
 
   <% @posts.each do |post| %>
-    <%= c.post(post: post) %>
+    <%= c.post { post } %>
```

Comment:
```suggestion
    <%= c.post do %>
      <%= link_to post.name, post.url %>
    <% end %>
```

Adding some markup in here might better communicate that this is a passthrough slot that'll render the given markup.